docs/user/explanation/working-with-code/git-hosting.rst

```diff
@@ -44,9 +44,7 @@ Launchpad supports Git hosting. This means that you can:
 What will be supported?
 -----------------------
 
-Launchpad's Bazaar support has grown many features over the years, and
-it will take some time to bring our Git support up to full parity with
-it. Here's an incomplete list of some of the features we expect to add:
+Here's an incomplete list of some of the features we expect to add:
```

Comment:
I'm not sure we should have this paragraph at all tbh - I'd remove this "What will be supported?" section altogether instead of re-working it

There are a lot of features we 'd love to add besides these, but there isn't much in our current roadmap to add these specifically